delta/utils/loss/loss_utils.py

```diff
@@ -240,27 +241,28 @@ def arcface_loss(embedding,
         name='arcface_loss_output')
   return output
 
+
 def focal_loss(logits, labels, gamma=2, name='focal_loss'):
-    """
+  """
```

Comment:
It seems that most part in the repo uses 4 spaces for indentation. Why do you change to use 2 spaces? Either 4 or 2 is fine, as long as we make it consistent ( I personally prefer 4 though :-))